### PR TITLE
use KEDA v2.8.1 & ship v2.8.2

### PR DESCRIPTION
**Pull request by JorTurFer**
_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Related to https://github.com/kedacore/keda/issues/3626

### CODE REVIEW

File: keda/Chart.yaml [hunk 1]
```diff
@@ -8,11 +8,11 @@ kubeVersion: ">=v1.20.0-0"
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.8.1
+version: 2.8.2
```
Comment (tomkerkhove) [inline keda/Chart.yaml:11]: Please revert this as per https://github.com/kedacore/keda/issues/3626#issuecomment-1234109552

https://github.com/kedacore/charts/releases/tag/v2.8.1

**Review by tomkerkhove — CHANGES_REQUESTED** [review]

File: keda/Chart.yaml (hunk 1 above)
Comment (JorTurFer) [reply keda/Chart.yaml:11]: Wasn't your bump already released? https://github.com/kedacore/charts/commit/b0eb796ae604fda2b6492f3a693dd8382bd6955e#diff-22efdd45972c44fc1f4743550c58a1d82b372e7c2ba340c5ae89d832538aacbf

Comment (tomkerkhove) [reply keda/Chart.yaml:11]: You are correct, I thought this PR was already merged!
https://github.com/kedacore/charts/pull/312/files#diff-22efdd45972c44fc1f4743550c58a1d82b372e7c2ba340c5ae89d832538aacbfR11

We can merge it and change the readme before shipping unless you prefer to move on?

**Review by tomkerkhove — APPROVED** [review]

File: keda/Chart.yaml (hunk 1 above)
Comment (JorTurFer) [reply keda/Chart.yaml:11]: I'll do it

**Pull request merged**